posixlib/src/main/resources/scala-native/unistd.c

```diff
@@ -9,6 +13,16 @@ int scalanative_w_ok() { return W_OK; }
 
 int scalanative_x_ok() { return X_OK; }
 
+// SEEK_CUR, SEEK_END, SEEK_SET in clib stdio
```

Comment:
Again, IMO, those three should be defined here, probably as calls to the clib version.
Independent duplication is also possible, if you/we want to avoid a dependency of posixlib on clib.

Reply:
There is not specific dependency but I get the point - some of this discussed later.

Reply:
I looked at this and they were suppose to be in POSIX `stdio.scala` or `stdio.c` and there isn't one at all so this was the minimal fix that was actually correct.